src/queries/staking_queries.ts

```diff
@@ -322,9 +336,8 @@ export const nextEpochPoolsStatsQuery = `
             , cs.zrx_staked
             , ts.total_staked
             , cs.zrx_staked / ts.total_staked AS share_of_stake
-            , fbp.protocol_fees AS protocol_fees_generated_in_eth
-            , tr.total_protocol_fees
-            , fbp.protocol_fees / tr.total_protocol_fees AS share_of_fees
+            , 0.00 AS total_protocol_fees_generated_in_eth
+            , 0 AS number_of_fills
```

Comment:
Are these supposed to be hardcoded to `0`?

Reply:
This was intentional. These are for the next epoch so putting a non-zero value here is misleading I think.